### ISSUE TITLE
Add from icon ability to disable/enable extensions

### ISSUE THREAD

**Issue by davidshq** [davidshq/auto-extension-manager#7]
Chrome offers a nice extension icon in the browser and one can click on it to determine which extensions are pinned, what access specific extensions have to the current site, BUT it does not offer the ability to enable/disable the extensions from the icon.

Currently one has to go into the extensions browser section and enable/disable. This interrupts flow of work. Add the ability to disable/enable extensions directly from the auto extension manager icon (via "popup").

**Comment by davidshq** [comment]
A nice-to-have feature would be if one could use keyboard shortcuts to quickly enable/disable specific extensions - but this shouldn't be part of this feature initially. Once this feature is done consider creating a new issue for the enhancement.
